lib/masamune/data_plan/engine.rb

```diff
@@ -121,6 +121,8 @@ def sources(rule)
   def prepare(rule, options = {})
     @targets[rule].merge options.fetch(:targets, [])
     @sources[rule].merge options.fetch(:sources, [])
+    @target_rules[rule].try(:prepare)
+    @source_rules[rule].try(:prepare)
```

Comment:
forces eager load of `fs.path()` before execution so that plan fails faster if `fs.path` is undefined